### DISCUSSION TITLE
January Release - New Comments UI, Improved Diff View, and more

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#33] [Announcements]
Hello everyone! We've been busy over the holiday season, and spent a lot of time fixing critical bugs, improving performance, and adding key features.

### Improved Comments UI
We've looked into the way comments are being used across GitBook, and have made some major improvements not only to the way they function, but also to the way they allow you to work with the rest of your team. Here's a quick summary of some of the major changes:

- Comments moved to sidebar instead of overlay
- Context added to block level comments
- Comments available in diff mode

<img width="1148" alt="Screenshot 2023-01-24 at 15 06 51" src="https://user-images.githubusercontent.com/42930383/214316763-9a2cd8af-2a9a-442f-b67a-26595db864b8.png">

### Updated diff view & indicators
We've also made some major UX improvements to the way diff view looks and feels, allowing you to have greater context to the the things that have changed inside a document you might be working on.

- Deleted blocks now show a diff indicator in the editor
- Apply new diff view UI elements in diff mode
- Added modified/added diff indicators when viewing a change request

<img width="1435" alt="Screenshot 2023-01-24 at 15 05 23" src="https://user-images.githubusercontent.com/42930383/214316975-45435aab-1ce5-4248-b952-cfb7c55a10e0.png">

### OpenAPI request body examples added
If you're using the OpenAPI block in any of your GitBook pages, you'll now see any request body examples included in the definition file used to generate the block.

### Notifications are now sent for all space reviewers for change requests
Lastly, we've improved the way notifications are shared when change requests are finished and submitted for review.

---

In addition to the updates above, we've fixed many bugs and added improvements across the app. Make sure you check out the [entire changelog](https://changelog.gitbook.com/january-2023) to see everything that's changed.

**Comment by JaniceManwiller** [comment]
I'm not sure the new diff markings are working correctly. In my current doc CRs, there is a green plus sign with a "this block was added by this change request" hover message for every single block in every single topic, including topics that have no changes at all. If I toggle Diff View on, it shows the correct changes.

**Reply by addisonschultz** [reply]
Hi @JaniceManwiller! Thanks for letting us know - we see it on our end too, and already have a hot-fix deploying as we speak. Hang tight!
